tests/core/Jalasoft.TeamUp.Resumes.Core.Tests/UpdateResumeCoreTests.cs

```diff
@@ -42,9 +44,21 @@ public static List<Skill> GetResume()
         [Fact]
         public void UpdateResume_ExistentId_Resume()
         {
-            this.mockResumeRepository.Setup(repository => repository.UpdateResumeSkill(It.IsAny<int>(), It.IsAny<Skill[]>())).Returns(UpdateResumeCoreTests.GetResume());
-            var result = this.resumesService.UpdateResumeSkill(1, UpdateResumeCoreTests.GetResume().ToArray());
+            this.mockResumeRepository.Setup(repository => repository.UpdateResumeSkill(It.IsAny<int>(), It.IsAny<Skill[]>())).Returns(UpdateResumeCoreTests.GetSkills());
+            this.mockResumeRepository.Setup(repository => repository.GetById(It.IsAny<int>())).Returns(new Resume());
+            var result = this.resumesService.UpdateResumeSkill(2, UpdateResumeCoreTests.GetSkills().ToArray());
             Assert.IsType<Skill[]>(result.ToArray());
         }
+
+        [Fact]
+        public void UpdateResume_UnexistentId_ValidationException()
+        {
+            Resume resume = null;
+            var error = new ValidationFailure("Resume", "Object Doesn't exist");
+            error.ErrorCode = "404";
+            this.mockResumeRepository.Setup(repository => repository.UpdateResumeSkill(It.IsAny<int>(), It.IsAny<Skill[]>())).Throws(new ValidationException(new List<ValidationFailure>() { error }));
```

Comment:
the DAL does not return a validation exception this is done in the service, this mock should be done to the getbyId method and return a null so that the validation exception is thrown